kpatch-build/kpatch-elf.h

```diff
@@ -131,6 +131,22 @@ struct rela *find_rela_by_offset(struct section *relasec, unsigned int offset);
 
 int offset_of_string(struct list_head *list, char *name);
 
+#ifdef __powerpc__
+#ifndef R_PPC64_ENTRY
+#define R_PPC64_ENTRY	118
+#endif
+
+/* Symbol st_others value for powerpc */
+#define STO_PPC64_LOCAL_BIT     5
+#define STO_PPC64_LOCAL_MASK    (7 << STO_PPC64_LOCAL_BIT)
+#define PPC64_LOCAL_ENTRY_OFFSET(other)                                 \
+	(((1 << (((other) & STO_PPC64_LOCAL_MASK) >> STO_PPC64_LOCAL_BIT)) >> 2) << 2)
```

Comment:
I'd rather have these defines for st_other to be in the .c file, right above the function which uses them.